<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
        <item>
         <widget class="QTreeView" name="treeFileView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>186</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>186</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QMdiArea" name="mdiArea"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,13,1">
      <item>
       <widget class="QLineEdit" name="enter_filename_search">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>186</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>186</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_file_path">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Search_file">
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="create"/>
    <addaction name="open"/>
    <addaction name="openReadOnly"/>
    <addaction name="save"/>
    <addaction name="print"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="options">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="control">
     <property name="title">
      <string>Управление</string>
     </property>
     <addaction name="buttons"/>
    </widget>
    <widget class="QMenu" name="language">
     <property name="title">
      <string>Язык</string>
     </property>
     <addaction name="russian"/>
     <addaction name="english"/>
    </widget>
    <widget class="QMenu" name="theme">
     <property name="title">
      <string>Тема</string>
     </property>
     <addaction name="light"/>
     <addaction name="dark"/>
    </widget>
    <addaction name="control"/>
    <addaction name="language"/>
    <addaction name="theme"/>
   </widget>
   <widget class="QMenu" name="faq">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutProgram"/>
   </widget>
   <widget class="QMenu" name="view">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="treeViewShow"/>
   </widget>
   <widget class="QMenu" name="edit">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="Rand_color"/>
    <addaction name="Align_left"/>
    <addaction name="Align_center"/>
    <addaction name="Align_right"/>
    <addaction name="Font"/>
    <addaction name="CopyFormat"/>
    <addaction name="PasteFormat"/>
    <addaction name="Enter_Date"/>
    <addaction name="Enter_time"/>
   </widget>
   <addaction name="file"/>
   <addaction name="edit"/>
   <addaction name="view"/>
   <addaction name="options"/>
   <addaction name="faq"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="create"/>
   <addaction name="open"/>
   <addaction name="save"/>
   <addaction name="print"/>
   <addaction name="Rand_color"/>
   <addaction name="Align_left"/>
   <addaction name="Align_center"/>
   <addaction name="Align_right"/>
   <addaction name="Font"/>
   <addaction name="CopyFormat"/>
   <addaction name="PasteFormat"/>
   <addaction name="Enter_Date"/>
   <addaction name="Enter_time"/>
  </widget>
  <action name="open">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/icons8-opened-folder-64.png</normaloff>:/image/png/image/icon/icons8-opened-folder-64.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="openReadOnly">
   <property name="text">
    <string>Открыть для чтения</string>
   </property>
  </action>
  <action name="save">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/icons8-save-64.png</normaloff>:/image/png/image/icon/icons8-save-64.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="buttons">
   <property name="text">
    <string>Клавиши</string>
   </property>
  </action>
  <action name="russian">
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="english">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="light">
   <property name="text">
    <string>Светлая</string>
   </property>
  </action>
  <action name="dark">
   <property name="text">
    <string>Тёмная</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="create">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/icons8-new-copy-64.png</normaloff>:/image/png/image/icon/icons8-new-copy-64.png</iconset>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="aboutProgram">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="treeViewShow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Проводник</string>
   </property>
  </action>
  <action name="print">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/icons8-print-64.png</normaloff>:/image/png/image/icon/icons8-print-64.png</iconset>
   </property>
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
  <action name="Rand_color">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/icons8-маркер-64.png</normaloff>:/image/png/image/icon/icons8-маркер-64.png</iconset>
   </property>
   <property name="text">
    <string>Выделить цветом</string>
   </property>
  </action>
  <action name="Align_center">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/icons8-выровнять-по-центру-64.png</normaloff>:/image/png/image/icon/icons8-выровнять-по-центру-64.png</iconset>
   </property>
   <property name="text">
    <string>По центру</string>
   </property>
  </action>
  <action name="Align_left">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/icons8-выровнять-по-левому-краю-64.png</normaloff>:/image/png/image/icon/icons8-выровнять-по-левому-краю-64.png</iconset>
   </property>
   <property name="text">
    <string>По левому краю</string>
   </property>
   <property name="toolTip">
    <string>По левому краю</string>
   </property>
  </action>
  <action name="Align_right">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/icons8-выровнять-по-правому-краю-64.png</normaloff>:/image/png/image/icon/icons8-выровнять-по-правому-краю-64.png</iconset>
   </property>
   <property name="text">
    <string>По правому краю</string>
   </property>
   <property name="toolTip">
    <string>По правому краю</string>
   </property>
  </action>
  <action name="Font">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/icons8-выбор-шрифта-64.png</normaloff>:/image/png/image/icon/icons8-выбор-шрифта-64.png</iconset>
   </property>
   <property name="text">
    <string>Шрифт</string>
   </property>
  </action>
  <action name="CopyFormat">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/icons8-поставить-ссылку-на-родительскую-компанию-64.png</normaloff>:/image/png/image/icon/icons8-поставить-ссылку-на-родительскую-компанию-64.png</iconset>
   </property>
   <property name="text">
    <string>Копировать формат</string>
   </property>
   <property name="toolTip">
    <string>Копировать формат</string>
   </property>
  </action>
  <action name="PasteFormat">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/icons8-поставить-ссылку-на-дочернюю-компанию-64.png</normaloff>:/image/png/image/icon/icons8-поставить-ссылку-на-дочернюю-компанию-64.png</iconset>
   </property>
   <property name="text">
    <string>Вставить формат</string>
   </property>
   <property name="toolTip">
    <string>Вставить формат</string>
   </property>
  </action>
  <action name="Enter_Date">
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/eventonday10_79977.png</normaloff>:/image/png/image/icon/eventonday10_79977.png</iconset>
   </property>
   <property name="text">
    <string>Вставить дату</string>
   </property>
  </action>
  <action name="Enter_time">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="OtherRes.qrc">
     <normaloff>:/image/png/image/icon/clock_icon_128908.png</normaloff>:/image/png/image/icon/clock_icon_128908.png</iconset>
   </property>
   <property name="text">
    <string>Вставить время</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="OtherRes.qrc"/>
 </resources>
 <connections/>
</ui>
